src/mame/skeleton/webtouchone.cpp

```diff
@@ -0,0 +1,158 @@
+// license:BSD-3-Clause
+// copyright-holders:
+
+/*
+  Skeleton driver for Alcatel Web Touch One.
+  More information and technical manuals: https://www.minitel-alcatel.fr/gamme_webphone.html
+  The phone asks for an adminstrative password for performing a factory reset.
+
+  Hardware info for model 2840:
+   -CPU: Motorola Power PC 823
+   -Operating system: JavaOS over ChorusOS (Sun Microsystems)
+   -Screen: Hitachi SX19V001-ZZA (7,6'' touch, color, 640x480)
+   -Keyboard: N860-1428-T021
+   -Other: Smartcard reader, 33.6 kbps modem (Conexant RVC3366ACFW), 8MB DRAM, etc.
+
+  CPU PCB
+    ____________________________________________________________________
+   |        Xtal                                                       |
+   |     25.416 MHz                                                    |
+   |    _____________                         _______                  |
+   |   | CONEXANT   |                        |MC3403|   ___            |
+   |   | RVC3366ACFW|                                  |  |            |
+   |   | R6749-24   |   ________  ________   74HC4053->|  |            |
+   |   |            |  |74HC4053||74HC4053|            |__|            |
+   |   |____________|                    ___                   _______ |
+   |  _______                    34119->|  |                  |MC3403| |
+   | 74AHCT574	                        |__|                           |
+   |  ____________   ____________                                      |
+   | |KM416S4030CT| |KM416S4030CT|                                     |
+   | |____________| |____________|                                     |
+   |       ____________                                     __________ |
+   |      |KM29U64000T|             _____________          |74HCT541 | |
+   |      |___________|            | Motorola   |          |_________| |
+   |                               | XPC823ZT66A|           __________ |
+   |                    Xtal       |            |          |74HCT541 | |
+   |                 32.768 MHz    |            |          |_________| |
+   |                               |____________|                      |
+   |                                                                   | 
+   |                                          _______                  |
+   | 3BN62121AAAF KAZZA 01                   |ST 324|                  |
+   |___________________________________________________________________|
+
```

Comment:
Legit puzzled about what RAMDAC this uses, MPC823 datasheet mentions having one for the LCD interface. Where's TFT connector?

Reply:
Cannot remember, I dumped it four years ago, and I only keep photos of the disconnected PCBs.
There is some more hardware info here: https://www.minitel-alcatel.fr/documents/WebPhones/SES%20Hard%202840.pdf